platform/qemu.go

```diff
@@ -351,3 +366,93 @@ func Virtio(board, device, args string) string {
 	}
 	return fmt.Sprintf("virtio-%s-%s,%s", device, suffix, args)
 }
+
+func setupIgnition(confPath string, diskImagePath string) error {
+	fileRemoteLocation := "/boot/ignition/config.ign"
+
+	plog.Debugf("START guestfs")
+
+	cmd := exec.Command("guestfish", "--listen", "-a", diskImagePath)
+	stdout, err := cmd.StdoutPipe()
```

Comment:
@arithx I tried to substitute `cmd.Output()` to `cmd.StdoutPipe()` and it didn't wokr. I think the right choice is `cmd.StdoutPipe()` because guestfish is started in background and the `cmd.Output` returns only when the comand finish.